src/deadline/houdini_adaptor/HoudiniClient/houdini_handler.py

```diff
@@ -44,30 +44,17 @@ def set_node_settings(self, node):
         node_type = node.type().nameWithCategory().split("/")
         if node_type[0] == "Driver":
             if node_type[1] == "ifd":
-                # mantra render node
+                # Mantra render node
                 alfredProgress = node.parm("vm_alfprogress")
                 if alfredProgress is not None:
                     alfredProgress.set(1)
                     print("Enabled Alfred style progress")
-
-                verbosity = node.parm("vm_verbose")
-                if verbosity is not None:
-                    verbosity.set(3)
-                    print("Set verbosity to 3")
```

Comment:
Is this the right change for the issue? It makes sense to me that we should have a reasonable, minimum verbosity set for these nodes to ensure we have _some_ output in the render logs.

The reason this code was wrong previously, was that we could not _increase_ the verbosity due to always setting it to 3 which prevented the ability to further debug the scene.

Reply:
After this is merged in, the behaviour is as follows:
1. We always report progress from 0 to 100 (due to `vm_alfprogress` or `alfprogress`)
2. We follow the customer's configured log verbosity, whether it is zero or max

I think this is the right behaviour and addresses your concerns. It provides _some_ logs by default (the progress), but allows customers the greatest flexibility to configure their own settings by default. I did run this with various choices of log verbosity from both Mantra and Karma and they are indeed followed.

Reply:
Mantra logs with verbosity 0:
```
2024/05/03 12:07:56-05:00 ADAPTOR_OUTPUT: STDOUT: Enabled Alfred style progress
2024/05/03 12:07:57-05:00 ADAPTOR_OUTPUT: STDOUT: 17:07:56  mantra1 frame 5
2024/05/03 12:08:11-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 0%
2024/05/03 12:08:17-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 1%
2024/05/03 12:08:17-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 2%
2024/05/03 12:08:17-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 3%
2024/05/03 12:08:18-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 4%
2024/05/03 12:08:19-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 5%
2024/05/03 12:08:20-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 6%
2024/05/03 12:08:20-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 7%
2024/05/03 12:08:21-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 8%
2024/05/03 12:08:21-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 9%
2024/05/03 12:08:22-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 10%
2024/05/03 12:08:23-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 11%
2024/05/03 12:08:23-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 12%
2024/05/03 12:08:24-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 13%
2024/05/03 12:08:25-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 14%
2024/05/03 12:08:25-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 15%
2024/05/03 12:08:26-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 16%
2024/05/03 12:08:27-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 17%
2024/05/03 12:08:27-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 18%
2024/05/03 12:08:28-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 19%
2024/05/03 12:08:29-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 20%
2024/05/03 12:08:29-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 21%
2024/05/03 12:08:30-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 22%
2024/05/03 12:08:31-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 23%
2024/05/03 12:08:32-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 24%
2024/05/03 12:08:32-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 25%
2024/05/03 12:08:33-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 26%
2024/05/03 12:08:34-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 27%
2024/05/03 12:08:36-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 28%
2024/05/03 12:08:36-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 29%
2024/05/03 12:08:38-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 30%
2024/05/03 12:08:39-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 31%
2024/05/03 12:08:40-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 32%
2024/05/03 12:08:40-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 33%
2024/05/03 12:08:41-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 34%
2024/05/03 12:08:42-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 35%
2024/05/03 12:08:43-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 36%
2024/05/03 12:08:44-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 37%
2024/05/03 12:08:45-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 38%
2024/05/03 12:08:46-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 39%
2024/05/03 12:08:47-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 40%
2024/05/03 12:08:49-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 41%
2024/05/03 12:08:49-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 42%
2024/05/03 12:08:51-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 43%
2024/05/03 12:08:51-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 44%
2024/05/03 12:08:53-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 45%
2024/05/03 12:08:53-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 46%
2024/05/03 12:08:55-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 47%
2024/05/03 12:08:55-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 48%
2024/05/03 12:08:56-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 49%
2024/05/03 12:08:58-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 50%
2024/05/03 12:08:59-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 51%
2024/05/03 12:09:00-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 52%
2024/05/03 12:09:00-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 53%
2024/05/03 12:09:01-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 54%
2024/05/03 12:09:02-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 55%
2024/05/03 12:09:03-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 56%
2024/05/03 12:09:04-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 57%
2024/05/03 12:09:05-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 58%
2024/05/03 12:09:06-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 59%
2024/05/03 12:09:07-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 60%
2024/05/03 12:09:08-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 61%
2024/05/03 12:09:09-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 62%
2024/05/03 12:09:10-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 63%
2024/05/03 12:09:11-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 64%
2024/05/03 12:09:11-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 65%
2024/05/03 12:09:12-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 66%
2024/05/03 12:09:13-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 67%
2024/05/03 12:09:14-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 68%
2024/05/03 12:09:15-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 69%
2024/05/03 12:09:16-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 70%
2024/05/03 12:09:18-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 71%
2024/05/03 12:09:18-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 72%
2024/05/03 12:09:20-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 73%
2024/05/03 12:09:20-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 74%
2024/05/03 12:09:21-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 75%
2024/05/03 12:09:22-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 76%
2024/05/03 12:09:22-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 77%
2024/05/03 12:09:23-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 78%
2024/05/03 12:09:24-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 79%
2024/05/03 12:09:24-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 80%
2024/05/03 12:09:25-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 81%
2024/05/03 12:09:26-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 82%
2024/05/03 12:09:26-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 83%
2024/05/03 12:09:27-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 84%
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 85%
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 86%
2024/05/03 12:09:29-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 87%
2024/05/03 12:09:29-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 88%
2024/05/03 12:09:30-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 89%
2024/05/03 12:09:31-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 90%
2024/05/03 12:09:31-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 91%
2024/05/03 12:09:32-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 92%
2024/05/03 12:09:33-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 93%
2024/05/03 12:09:33-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 94%
2024/05/03 12:09:34-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 95%
2024/05/03 12:09:34-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 96%
2024/05/03 12:09:35-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 97%
2024/05/03 12:09:36-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 98%
2024/05/03 12:09:36-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 99%
2024/05/03 12:09:37-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 100%
2024/05/03 12:09:37-05:00 ADAPTOR_OUTPUT: STDOUT: Finished Rendering
2024/05/03 12:09:37-05:00 INFO: Done HoudiniAdaptor main
```

Mantra verbosity  1 adds:
```

2024/05/03 12:07:54-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:53] Generating Image: /sessions/session-6c4a5e193da44674a7ff51250f45aec5jswby2ps/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.mantra1.0005.png (1280x720)
2024/05/03 12:07:54-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:53]  Plane[C]: 0 Cf+Af[4] (16-bit float)
...
2024/05/03 12:09:31-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:30] Frame Wall Clock Time: 0:01:37.87
2024/05/03 12:09:31-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:30] Total Wall Clock Time: 0:01:37.87
2024/05/03 12:09:31-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:30]        Total CPU Time: 0:02:58.29
2024/05/03 12:09:31-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:30]  System CPU Time Only: 0:00:02.14
2024/05/03 12:09:31-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:30]     Peak Memory Usage: 385.31 MB
```


Mantra verbosity 2 adds:
```
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27]   reading geometry from /tmp/houdini_temp/ifds/storage/2483_animatedj.5_000_0060.bgeo.sc
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27] /obj/geo1/out(loading...)
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27]   reading geometry from /tmp/houdini_temp/ifds/storage/2483_animatedj.5_000_0085.bgeo.sc
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27] /obj/grid1/grid1(loading...)
...
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27]     SampleFilter: alpha
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27]     PixelFilter:  gaussian -w 2
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27]     VEX Type:     vector4
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27]     Gamma:        1
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:09:27] Load Time:
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: Frame Wall Clock Time: 0:00:00.58
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT: Total Wall Clock Time: 0:00:00.58
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT:        Total CPU Time: 0:00:01.81
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT:  System CPU Time Only: 0:00:00.36
2024/05/03 12:09:28-05:00 ADAPTOR_OUTPUT: STDOUT:     Peak Memory Usage: 81.53 MB
...
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16] Ray Counts:
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16]       8,330,436 primary rays          9.04 rays/pixel
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16]      18,348,851 occlusion rays       19.91 rays/pixel
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16]      43,570,591 opacity rays         47.28 rays/pixel
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16]      44,526,639 shading rays         48.31 rays/pixel
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16]     114,776,517 total rays          124.54 rays/pixel
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16] Unified Cache: 24.00 MB of 1.91 GB used
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16]                       In Cache  Faults    Generated    Item Size
2024/05/03 12:11:16-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:16]      TBF Texture:     24.00 MB    3160     24.00 MB      7.78 KB
```

Mantra verbosity 3 adds:
```

2024/05/03 12:10:13-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:13] 
2024/05/03 12:10:13-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:13] Thread Count: 2
2024/05/03 12:10:20-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:20] Creating geometry (/obj/geo1)
2024/05/03 12:10:25-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:25] Building octree
...
2024/05/03 12:11:58-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:58] TBF File Cache: 2 file opens (6.88 MB read)
2024/05/03 12:11:58-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:58] 	1 non-RAT image conversions using 16.00 MB
```

Mantra verbosity 4 adds:
```

2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] User defined plane attribute 'IPlay.s3dleftplane' 'string'[1]
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] User defined plane attribute 'IPlay.s3drightplane' 'string'[1]
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] User defined plane attribute 'IPlay.rendermode' 'string'[1]
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] User defined plane attribute 'IPlay.framerange' 'string'[1]
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] User defined plane attribute 'IPlay.currentframe' 'float'[1]
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] User defined plane attribute 'IPlay.rendersource' 'string'[1]
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] User defined plane attribute 'IPlay.houdiniportnum' 'int'[1]
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] User defined plane attribute 'IPlay.label' 'string'[1]
...
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] Will be writing to checkpoint file /sessions/session-d4f0f713c6874a0e813dddc7e55255daple59d3j/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.mantra1.0005.png.mantra_checkpoint
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] Deleting temporary geometry '/tmp/houdini_temp/ifds/storage/2485_animatedj.5_000_0060.bgeo.sc'
2024/05/03 12:10:14-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:14] Deleting temporary geometry '/tmp/houdini_temp/ifds/storage/2485_animatedj.5_000_0085.bgeo.sc'
...
2024/05/03 12:11:55-05:00 ADAPTOR_OUTPUT: STDOUT: [17:11:55] Threading Efficiency: 199.88%
```


Karma logs with verbosity "None":
```
2024/05/02 16:17:05-05:00 ADAPTOR_OUTPUT: STDOUT: 21:17:04  rop_usdrender frame 5
2024/05/02 16:17:09-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 0%
2024/05/02 16:17:12-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 0%
2024/05/02 16:17:17-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 1%
2024/05/02 16:17:20-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 2%
2024/05/02 16:17:21-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 3%
2024/05/02 16:17:23-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 4%
2024/05/02 16:17:25-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 5%
2024/05/02 16:17:26-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 6%
2024/05/02 16:17:28-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 7%
2024/05/02 16:17:30-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 8%
2024/05/02 16:17:31-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 9%
2024/05/02 16:17:33-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 10%
2024/05/02 16:17:34-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 11%
2024/05/02 16:17:36-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 12%
2024/05/02 16:17:38-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 13%
2024/05/02 16:17:40-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 14%
2024/05/02 16:17:42-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 15%
2024/05/02 16:17:43-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 16%
2024/05/02 16:17:45-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 17%
2024/05/02 16:17:47-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 18%
2024/05/02 16:17:48-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 19%
2024/05/02 16:17:50-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 20%
2024/05/02 16:17:51-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 21%
2024/05/02 16:17:53-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 22%
2024/05/02 16:17:55-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 23%
2024/05/02 16:17:57-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 24%
2024/05/02 16:17:59-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 25%
2024/05/02 16:18:00-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 26%
2024/05/02 16:18:02-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 27%
2024/05/02 16:18:04-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 28%
2024/05/02 16:18:05-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 29%
2024/05/02 16:18:07-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 30%
2024/05/02 16:18:09-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 31%
2024/05/02 16:18:10-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 32%
2024/05/02 16:18:12-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 33%
2024/05/02 16:18:14-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 34%
2024/05/02 16:18:16-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 35%
2024/05/02 16:18:18-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 36%
2024/05/02 16:18:19-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 37%
2024/05/02 16:18:21-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 38%
2024/05/02 16:18:22-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 39%
2024/05/02 16:18:24-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 40%
2024/05/02 16:18:26-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 41%
2024/05/02 16:18:27-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 42%
2024/05/02 16:18:29-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 43%
2024/05/02 16:18:30-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 44%
2024/05/02 16:18:32-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 45%
2024/05/02 16:18:34-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 46%
2024/05/02 16:18:36-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 47%
2024/05/02 16:18:38-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 48%
2024/05/02 16:18:39-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 49%
2024/05/02 16:18:41-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 50%
2024/05/02 16:18:42-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 51%
2024/05/02 16:18:44-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 52%
2024/05/02 16:18:45-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 53%
2024/05/02 16:18:47-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 54%
2024/05/02 16:18:48-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 55%
2024/05/02 16:18:50-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 56%
2024/05/02 16:18:52-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 57%
2024/05/02 16:18:54-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 58%
2024/05/02 16:18:56-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 59%
2024/05/02 16:18:57-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 60%
2024/05/02 16:18:59-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 61%
2024/05/02 16:19:01-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 62%
2024/05/02 16:19:02-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 63%
2024/05/02 16:19:04-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 64%
2024/05/02 16:19:05-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 65%
2024/05/02 16:19:07-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 66%
2024/05/02 16:19:08-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 67%
2024/05/02 16:19:10-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 68%
2024/05/02 16:19:12-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 69%
2024/05/02 16:19:14-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 70%
2024/05/02 16:19:16-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 71%
2024/05/02 16:19:17-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 72%
2024/05/02 16:19:19-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 73%
2024/05/02 16:19:20-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 74%
2024/05/02 16:19:22-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 75%
2024/05/02 16:19:23-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 76%
2024/05/02 16:19:25-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 77%
2024/05/02 16:19:26-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 78%
2024/05/02 16:19:28-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 79%
2024/05/02 16:19:30-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 80%
2024/05/02 16:19:32-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 81%
2024/05/02 16:19:34-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 82%
2024/05/02 16:19:35-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 83%
2024/05/02 16:19:37-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 84%
2024/05/02 16:19:38-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 85%
2024/05/02 16:19:40-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 86%
2024/05/02 16:19:41-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 87%
2024/05/02 16:19:43-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 88%
2024/05/02 16:19:44-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 89%
2024/05/02 16:19:46-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 90%
2024/05/02 16:19:48-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 91%
2024/05/02 16:19:50-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 92%
2024/05/02 16:19:52-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 93%
2024/05/02 16:19:53-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 94%
2024/05/02 16:19:55-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 95%
2024/05/02 16:19:56-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 96%
2024/05/02 16:19:58-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 97%
2024/05/02 16:19:59-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 98%
2024/05/02 16:20:01-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 99%
2024/05/02 16:20:02-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 100%
2024/05/02 16:20:02-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 100%
2024/05/02 16:20:03-05:00 ADAPTOR_OUTPUT: STDOUT: Finished Rendering
```

Karma verbosity "Rendering statistics" adds:
```

2024/05/03 12:07:17-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:17] husk version 19.5.569
2024/05/03 12:07:18-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:17] Setting default OIIO config: OPENIMAGEIO_IMAGECACHE_OPTIONS=max_memory_MB=3916,max_open_files=32767,autotile=512,autoscanline=1
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:18] Loading /tmp/houdini_temp/usd_renders/usdrender_2371_187_1/__render__.usd
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:19] Running prerender procedurals script: /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/.env/opt/houdini/houdini/python3.9libs/husd/runprocedurals.py --mode prerender --validategeo
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:19] Load 527 properties from /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/.env/opt/houdini/houdini/RenderProperties.json
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:19] Creating karma scene
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:19] Creating output directory: /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:19] Running preframe procedurals script: /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/.env/opt/houdini/houdini/python3.9libs/husd/runprocedurals.py --mode preframe --validategeo
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:19] Start Wall Clock Time: 0:00:01.73
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:19]   Start Memory Usage: 189.14 MB
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:19] >>> Render /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.karma1.0005.png, /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.karma1.0005.checkpoint
2024/05/03 12:07:19-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 0%
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Adding samplefilter module: copyalpha
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Pixel oracle: variance
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Load Wall Clock Time: 0:00:04.88
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Update Wall Clock Time: 0:00:03.20
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22]        Load CPU Time: 0:00:05.00
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Load System CPU Time: 0:00:00.57
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22]    Load Memory Usage: 485.45 MB
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Light Tree: 0 lights
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Use verbosity 6 to see reason lights weren't added
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Uniform List: 2 lights
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22] Acceleration build time (wall clock): 0:00.00
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 0%
2024/05/03 12:07:22-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:22]   0% Lap=0:00:04.98 Left=<unknown> Mem=241.48 MB Peak=241.48 MB
2024/05/03 12:07:26-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:25] First Pixel: 0:06.60
2024/05/03 12:07:28-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 1%
2024/05/03 12:07:28-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:27]   1% Lap=0:00:10.31 Left=0:16:47.99 Mem=262.18 MB Peak=266.16 MB
2024/05/03 12:07:30-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 2%
2024/05/03 12:07:30-05:00 ADAPTOR_OUTPUT: STDOUT: [17:07:29]   2% Lap=0:00:12.42 Left=0:10:08.56 Mem=263.18 MB Peak=266.16 MB
...
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Unified Cache: 24.00 MB of 1.91 GB used
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]                       In Cache  Faults    Generated    Item Size
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]      TBF Texture:     24.00 MB    3160     24.00 MB      7.78 KB
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] TBF File Cache: 2 file opens (6.88 MB read)
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Object Counts
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]        Polygons: 14,481
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]          Curves: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]           Point: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]    Point Meshes: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]         Volumes: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]     Procedurals: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]          Lights: 2
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]         Cameras: 1
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]        CoordSys: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Ray Counts
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]         primary: 8,147,903
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]        indirect: 74,378,286
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]       occlusion: 56,731,141
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]       light geo: 46,283,518
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]           probe: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]           total: 185,540,848
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Shader Calls
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]    Displacement: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]         Surface: 9,183,640
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]         Opacity: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]        Emission: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]           Light: 101,742,555
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]          Volume: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Frame Wall Clock Time: 0:02:49.81
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Total Wall Clock Time: 0:02:54.69
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]        Total CPU Time: 0:05:35.30
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]  System CPU Time Only: 0:00:03.08
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]  Current Memory Usage: 263.75 MB
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12]     Peak Memory Usage: 268.09 MB
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] VEX module cache misses: 0
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Saved Image: /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.karma1.0005.png
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Running postframe procedurals script: /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/.env/opt/houdini/houdini/python3.9libs/husd/runprocedurals.py --mode postframe --validategeo
2024/05/03 12:10:12-05:00 ADAPTOR_OUTPUT: STDOUT: [17:10:12] Running postrender procedurals script: /sessions/session-0b1b8673ecad4bbeb0a5a0ede02ea624fvtesqtb/.env/opt/houdini/houdini/python3.9libs/husd/runprocedurals.py --mode postrender --validategeo
2024/05/03 12:10:13-05:00 ADAPTOR_OUTPUT: STDOUT: Finished Rendering
2024/05/03 12:10:13-05:00 INFO: Done HoudiniAdaptor main
```

Karma logs with "Full verbosity":
```
2024/05/03 12:30:03-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:02] husk version 19.5.569
2024/05/03 12:30:03-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:02] Command Line: husk -s /Render/rendersettings --make-output-path -f 5 -R BRAY_HdKarma --purpose geometry,render --complexity veryhigh --snapshot 300 -V 9a /tmp/houdini_temp/usd_renders/usdrender_2312_187_1/__render__.usd
2024/05/03 12:30:03-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:02] CPU AMD EPYC 7571
2024/05/03 12:30:03-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:02] Architecture - AMD64: mmx xmmx sse sse2 cmov fpcmov ht sse3 ssse3 sse4.1 sse4.2 avx avx2
2024/05/03 12:30:03-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:02] System limit: 65535 file descriptors
2024/05/03 12:30:03-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:03] Setting default OIIO config: OPENIMAGEIO_IMAGECACHE_OPTIONS=max_memory_MB=3916,max_open_files=32767,autotile=512,autoscanline=1
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:03] Loading /tmp/houdini_temp/usd_renders/usdrender_2312_187_1/__render__.usd
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Running prerender procedurals script: /sessions/session-c0efd1ae45df4bd98b311c2fa53f12b727hta9sq/.env/opt/houdini/houdini/python3.9libs/husd/runprocedurals.py --mode prerender --validategeo
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Loading render settings: /Render/rendersettings
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] /Render/rendersettings: Loading product: /Render/Products/renderproduct
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] /Render/Products/renderproduct: Loading render var: /Render/Products/Vars/Beauty
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] /Render/Products/renderproduct contains 1 render vars
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] /Render/rendersettings: Loading product: /Render/Products/checkpoint
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] /Render/Products/checkpoint: Loading render var: /Render/Products/Vars/ignored_checkpoint_rendervar
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] /Render/Products/checkpoint contains 1 render vars
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] /Render/rendersettings contains 2 render products
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Load 527 properties from /sessions/session-c0efd1ae45df4bd98b311c2fa53f12b727hta9sq/.env/opt/houdini/houdini/RenderProperties.json
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04]  Imported 0 global defaults
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04]  Imported 0 object defaults
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04]  Imported 0 light defaults
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04]  Imported 1 camera defaults
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04]  Imported 0 plane defaults
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04]  Imported 0 command defaults
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Creating karma scene
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Initial Render Settings
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"karma:global:abortmissingtexture":false,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"karma:global:autoheadlight":true,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"karma:global:background_iprfilter":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"karma:global:bucketorder":"Middle",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"karma:global:bucketsize":32,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"karma:global:cacheratio":0.25,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"karma:global:cachesize":1000,
(hundreds more)

2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"materialBindingPurposes":["full","allPurpose"
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	],
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"pixelAspectRatio":1,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"renderCameraPath":"/cameras/cam1",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"renderMode":"batch",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"resolution":[1280,720],
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"shutter:close":0.25,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"shutter:open":-0.25,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"usdFileTimeStamp":1714757402,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"usdFilename":"/tmp/houdini_temp/usd_renders/usdrender_2312_187_1/__render__.usd",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"usdSettingsPrim":"/Render/rendersettings"
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT:     },
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT:     "RenderProducts":[
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    "settings":{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"driver:parameters:OpenEXR:compression":"zips",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"driver:parameters:artist":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"driver:parameters:comment":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"driver:parameters:hostname":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"productName":"/sessions/session-c0efd1ae45df4bd98b311c2fa53f12b727hta9sq/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.karma1.0005.png",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"productType":"raster",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"sourcePrim":"/Render/Products/renderproduct"
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    },
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    "expandedProductName":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    "RenderVariables":[
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "AOVName":"C",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "AOVPixelFormat":"float16",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "AOVChannelSize":4,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "HdFormat":"HdFormatFloat32Vec4",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "HdMultiSampled":true,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "HdClearValue":0,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "settings":{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"dataType":"color4f",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:channel_prefix":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:clearValue":0,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:cryptomatte":false,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:cryptomatterank":6,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:cryptomattesidecar":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:defaultvalue":0,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:dither":1,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:filter":"[\"ubox\",{}]",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:format":"color4h",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:multiSampled":true,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:name":"C",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:sampling":"average",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"sourceName":"C.*",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"sourcePrim":"/Render/Products/Vars/Beauty",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"sourceType":"lpe"
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    }
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		}
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    ]
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	},
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    "settings":{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"driver:parameters:OpenEXR:compression":"zips",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"driver:parameters:artist":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"driver:parameters:comment":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"driver:parameters:hostname":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"productName":"/sessions/session-c0efd1ae45df4bd98b311c2fa53f12b727hta9sq/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.karma1.0005.checkpoint",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"productType":"karma:checkpoint",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"resume":true,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"saveFrequency":60,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		"sourcePrim":"/Render/Products/checkpoint"
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    },
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    "expandedProductName":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    "RenderVariables":[
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "AOVName":"ignored_checkpoint_rendervar",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "AOVPixelFormat":"float32",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "AOVChannelSize":1,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "HdFormat":"HdFormatFloat32Vec3",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "HdMultiSampled":false,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "HdClearValue":0,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    "settings":{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"dataType":"color3f",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:channel_prefix":"",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:clearValue":0,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:format":"float",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:multiSampled":false,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"driver:parameters:aov:name":"ignored_checkpoint_rendervar",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"sourceName":"ignored_checkpoint",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"sourcePrim":"/Render/Products/Vars/ignored_checkpoint_rendervar",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"sourceType":"raw"
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		    }
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		}
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	    ]
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	}
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT:     ]
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: }
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Loading render settings: /Render/rendersettings
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] /Render/rendersettings contains 2 render products
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] SetRenderSetting[delegateRenderProducts] := [<'TfHashMap<TfToken, VtValue, TfToken::HashFunctor, equal_to<TfToken>, allocator<VtValue> >' @ 0x7f5efc331290>]
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Creating output directory: /sessions/session-c0efd1ae45df4bd98b311c2fa53f12b727hta9sq/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Running preframe procedurals script: /sessions/session-c0efd1ae45df4bd98b311c2fa53f12b727hta9sq/.env/opt/houdini/houdini/python3.9libs/husd/runprocedurals.py --mode preframe --validategeo
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Start Wall Clock Time: 0:00:01.50
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04]   Start Memory Usage: 189.14 MB
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] >>> Render /sessions/session-c0efd1ae45df4bd98b311c2fa53f12b727hta9sq/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.karma1.0005.png, /sessions/session-c0efd1ae45df4bd98b311c2fa53f12b727hta9sq/assetroot-b7592163465a901685f5/HoudiniTests/AnimatedJ/render/animatedj.karma1.0005.checkpoint
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] SetRenderSetting[randomseed] := 751277438
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: ALF_PROGRESS 0%
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Create HdSprim: camera /cameras/cam1
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Create HdRprim: mesh /geo_candidate/geo1/mesh_0
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Create HdSprim: material /mat/classicshader1
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Create HdRprim: mesh /geo_candidate/grid1/mesh_0
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Create HdSprim: material /mat/background
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Create HdSprim: domeLight /lights_cand/envlight2
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Create HdSprim: sphereLight /lights_cand/pointlight1
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Allocate AOV buffer: (1280, 720, 1) 15
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Sync camera /cameras/cam1
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Aspect ratio conform 0 H/V: 0.414214/0.232995, PAR: 1, IAR: 1.77778
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] 1 motion segments for /cameras/cam1
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Aspect ratio conform 1 H/V: 0.414214/0.232995, PAR: 1, IAR: 1.77778
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Camera /cameras/cam1:
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: {
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"transform":[[0.862045236317228847,-0.292305718441531193,0.414047554646991445,0,2.62443431237047164e-07,0.816933858976852112,0.576731367325481914,0,-0.50683134329153734,-0.497168419173592668,0.704234018230961989,0,0.29544066378419831,-0.720213957789915149,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			-12.9089494422292717,1]
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	],
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"projection":"perspective",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	"cameraProperties":[
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		{
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"focal":0.800000056624412537,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"aperture":0.232995361089706421,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"projection":1,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"lensshader":"RAW[ \"hou.node(ch('xn__karmacameralensshadervop_4fbg')).shaderString() if hou.node(ch('xn__karmacameralensshadervop_4fbg')) != None else ''\" python ]",
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"orthowidth":0.0232995361089706407,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"clip":[0.00100000004749745131,10000],
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"shutter":[-0.25,0.25],
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"focusdistance":37.4636192321777344,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"fstop":10.0666227340698242,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"window":[-1,1,-1,1],
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"exposure":0,
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 			"tint":[1,1,1]
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 		}
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 	]
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: }
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: 
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Sync Light /lights_cand/envlight2
2024/05/03 12:30:04-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Light /lights_cand/envlight2 xform[0]: ( (-1, 0, -1.2246467991473532e-16, 0), (0, 1, 0, 0), (1.2246467991473532e-16, 0, -1, 0), (0, 0, 0, 1) )
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_disney_brdf_2012_surface
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_disney_bsdf_2015_surface
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_oren_nayar_diffuse_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_burley_diffuse_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_translucent_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_dielectric_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_conductor_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_generalized_schlick_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_subsurface_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_sheen_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_thin_film_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_uniform_edf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_conical_edf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_measured_edf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_absorption_vdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_anisotropic_vdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_surface
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_thin_surface
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_volume
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_light
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_displacement_float
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_displacement_vector3
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_layer_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_layer_vdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_mix_bsdf
2024/05/03 12:30:05-05:00 ADAPTOR_OUTPUT: STDOUT: [17:30:04] Define shader node: ND_mix_edf
(hundreds more)
```

Reply:
Based on what we're seeing here (and knowing there are a few more verbosity options with more output), it seems like a reasonable minimum for the nodes when rendering in the service might be the following:
* Mantra: 2
* Karma: Render Statistics

The issue with 0 and None are that they don't give any indication of the output files being actually being rendered /input scene being read which are critical pieces of information when dealing with relocating files. It also doesn't pollute the log files which an immense amount of information as compared to the maximum verbosity settings.

In addition, it also turns out that on creation Mantra and Karma nodes default to 0 and None for verbosity so it seems like users will run into not having basic information pretty frequently.

Thoughts?

Reply:
Based on what we're seeing here (and knowing there are a few more verbosity options with more output), it seems like a reasonable minimum for the nodes when rendering in the service might be the following:

* Mantra: 2
* Karma: Render Statistics

The issue with 0 and None are that they don't give any indication of the output files being actually being rendered and input files being read which are critical pieces of information when dealing with relocating files. It also doesn't pollute the log files with an immense amount of information as compared to the maximum verbosity settings.

In addition, it also turns out that on creation Mantra and Karma nodes default to 0 and None for verbosity so it seems like users will run into not having basic information pretty frequently.

Thoughts?

Reply:
Discussed at length with Morgan. We feel it is better to include basic logs (Mantra 2 and Karma Rendering Statistics) for debugging by default rather than going with Houdini Mantra and Karma defaults of no logs. This will save customers time by reducing their debugging time. Instead of having to resubmit a job with a new verbosity setting, the customers will immediately have logs for debugging.